institutions_api/tests/test_app.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from fastapi.testclient import TestClient
+from institutions_api.app import app
+
+@pytest.fixture
+def api_client():
+    with TestClient(app) as client:
+        yield client
+
+class TestAPIEndpoints:
+
+    def test_get_valid_institutions(self, api_client):
+        """test whether getting a list of institutions works"""
+        response = api_client.get("/institution_ids")
+        assert response.status_code == 200
+        institutions = response.json()
+        assert isinstance(institutions, list)
+        assert len(institutions) > 0
+
+    def test_get_institution_details(self, api_client):
+        """test whether getting an institution details works"""
+        response = api_client.get("/institutions/3yiehdw3bef5")
+        assert response.status_code == 200
+        institution = response.json()
+        assert institution['name'] == "Academia Sinica"
+
+    def test_post_institution(self, api_client):
```

Comment:
For this test if I run it twice the second time it will fail because the name needs to be unique for each. 

Can you either add a random string to the end of the name or add a clean up after this test is ran to remove the Test institution?